### ISSUE TITLE
NoJS - Tablet/Phone navigation collapse doesn't work with No JS

### ISSUE THREAD

**Issue by nikdoof** [leigh-hackspace/website-hugo#47]
**Describe the bug**
The navigation collapse/expand button doesn't work without JS. On smaller screens/shrunk browsers this will make the menu inaccessible for NoJS browsers. Relates to #38 

**To Reproduce**
* Disable JS
* Shrink screen width under the tablet fold size (768px?)

**Expected behavior**
Still remain usable to some level with NoJS
